.gitignore

```diff
@@ -0,0 +1,2 @@
+*~
+go.sum
```

Comment:
It's always recommened to keep go.sum in your git repo, see https://github.com/golang/go/wiki/Modules#should-i-commit-my-gosum-file-as-well-as-my-gomod-file